package.json

```diff
@@ -1,57 +1,66 @@
 {
-    "name": "docs-inclusive-learning",
+    "name": "handbook.floeproject.org",
+    "author": "OCAD University",
+    "license": "BSD-3-Clause",
+    "version": "1.0.0",
     "description": "Strategies and best practices for inclusive learning content.",
-    "version": "0.1.0",
-    "author": "Floe Project",
-    "bugs": "https://github.com/fluid-project/docs-inclusive-learning/issues",
-    "repository": "https://github.com/fluid-project/docs-inclusive-learning",
-    "homepage": "https://handbook.floeproject.org/",
-    "dependencies": {
-        "@11ty/eleventy": "0.11.1",
-        "@11ty/eleventy-navigation": "0.1.6",
-        "@11ty/eleventy-plugin-syntaxhighlight": "3.0.4",
-        "@fluid-project/eleventy-plugin-fluid": "github:fluid-project/eleventy-plugin-fluid#ee75b78",
-        "docs-core": "fluid-project/docs-core"
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/fluid-project/handbook.floeproject.org.git"
     },
-    "devDependencies": {
-        "@commitlint/cli": "11.0.0",
-        "@commitlint/config-conventional": "11.0.0",
-        "cross-env": "7.0.2",
-        "debug": "4.2.0",
-        "fluid-grunt-lint-all": "2.0.0",
-        "grunt": "1.3.0",
-        "html-minifier": "4.0.0",
-        "husky": "4.3.0",
-        "infusion": "3.0.0-dev.20200326T173810Z.24ddb2718",
-        "jsdom": "16.4.0",
-        "laravel-mix": "5.0.7",
-        "move-file": "2.0.0",
-        "npm-run-all": "4.1.5",
-        "resolve-url-loader": "3.1.1",
-        "rimraf": "3.0.2",
-        "sass": "1.26.11",
-        "sass-loader": "8.0.2",
-        "slugify": "1.4.5",
-        "stylus": "0.54.8",
-        "stylus-loader": "3.0.2",
-        "vue-template-compiler": "2.6.12"
+    "bugs": {
+        "url": "https://github.com/fluid-project/handbook.floeproject.org/issues"
     },
-    "license": "BSD-3-Clause",
-    "keywords": [
-        "fluid",
-        "documentation",
-        "eleventy",
-        "static site",
-        "responsive"
-    ],
+    "homepage": "https://github.com/fluid-project/handbook.floeproject.org#readme",
     "scripts": {
         "build": "npm-run-all -l clean build:*",
-        "build:webpack": "cross-env NODE_ENV=production webpack --config=node_modules/laravel-mix/setup/webpack.config.js",
+        "build:webpack": "mix --production",
         "build:eleventy": "cross-env NODE_ENV=production eleventy",
         "clean": "rimraf dist",
-        "lint": "grunt lint",
+        "cms": "netlify-cms-proxy-server",
```

Comment:
Dependencies on `netlify-cms-proxy-server` can be removed too along with the removal of the Netlify CMS admin code.